src/tests/JNIGridnetClientSelfPlay.java

```diff
@@ -157,7 +157,6 @@ public byte[] render(boolean returnPixels) throws Exception {
     }
 
     public void gameStep(int[][] action1, int[][] action2) throws Exception {
-    	// FIXME do we really need to create this TraceEntry object at all?
```

Comment:
this FIXME comment was wrong. TraceEntry nicely wraps up exactly all the data that needs to passed into reward functions.